include/RAJA/policy/cuda/kernel/CudaKernel.hpp

```diff
@@ -301,8 +301,13 @@ struct CudaLaunchHelper<cuda_launch<async0, num_blocks, num_threads>,StmtList,Da
     }
   }
 
+  //
+  // actual_threads can be modified in the case where
+  // the requested number of theads would cause the
+  // max_blocks to be 0
+  //
   inline static void max_blocks(int shmem_size,
-      int &max_blocks, int actual_threads)
+      int &max_blocks, int &actual_threads)
   {
     auto func = internal::CudaKernelLauncher<Data, executor_t>;
```

Comment:
This should use the fixed kernel launch function if num_threads is greater than zero.
  internal::CudaKernelLauncherFixed<num_threads, Data, executor_t>
I broke this when I consolidated the CudaLaunchHelper implementation into a single code base.